packages/synapse-react-client/src/components/SynapsePriceFeatureRow.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Typography } from '@mui/material'
+import React, { useMemo } from 'react'
+import { SynapsePriceTableCell } from './SynapsePriceTableCell'
+import { CheckTwoTone } from '@mui/icons-material'
+
+export type SynapsePriceFeatureRowProps = {
+  featureDescription: string
+  standardPlanValue?: boolean | string
+  managedPlanValue?: boolean | string
+  dataCoordinationPlanValue?: boolean | string
+}
+
+export function getPriceFeatureCellContent(value?: boolean | string) {
```

Comment:
Prefer naming like a component using since this returns a ReactNode in all cases
```suggestion
export function PriceFeatureCellContent(props: {value?: boolean | string}) {
  const { value } = props
```